test/verification/test_avocado_stub_util.py

```
@@ -0,0 +1,13 @@
+from verification import avocado_stub_util
+
+
+def test_stub_renaming() -> None:
+    original_name_to_rename_metadata = avocado_stub_util.get_stub_mappings()
+    assert len(original_name_to_rename_metadata) != 0, f"Expected a non-zero number of stub function renamings"
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)